libbeat/docs/shared-command-line.asciidoc

```diff
@@ -14,7 +14,7 @@ Overwrite an individual setting in the config file. For example:
 +
 ["source","sh"]
 ----------------------------------------------------------------------
-sudo ./packetbeat -c packetbeat.yml -E name=mybeat
```

Comment:
This is used by all beats, not only packetbeat or metricbeat. 

@dedemorton Can we use a variable "inside" the code part?